src/registration.html

```diff
@@ -1,76 +1,92 @@
 <!DOCTYPE html>
 
-<html>
+<html class="fullsize">
     <head>
         <link rel="stylesheet" href="registration.css">
+        <link rel="preconnect" href="https://fonts.gstatic.com">
+        <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;700;900&display=swap" rel="stylesheet"> 
+        <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
     </head>
-    <body>
+    <body class="fullsize">
         <header>
-            vinyl exchange
+            <span class="material-icons">
+                <span class ="magenta">disc_full</span>
+            </span>
+            vinyl<span class ="magenta">exchange</span>
         </header>
-        <main>
+        <main class="fullsize">
             <form method="POST">
-                <div>
+                <div class="pad">
                     <label>
-                        First Name <input id="firstname" name="fname" type="text" required/>
+                        First Name 
+                        <p><input id="firstname" name="fname" class="inputstyle" type="text" required/></p>
```

Comment:
You are abusing `<p>` to break the line between the label and the input element. `<p>` is for denoting paragraphs in a text.
You could do multiple things here, one being that you make your `<input>`s siblings of their corresponding `<label>`s instead of their parent-child relationship and then specifying the parent `<div>`s as Flexbox containers.